apps/landing/features/home/HeadingSection/SignUpModal/index.tsx

```diff
@@ -127,20 +131,17 @@ const SignUpModal = () => {
 					text={email}
 					placeholder="zangimmortal@gmail.com"
 					onChangeText={handleChangeEmail}
+					onFocus={() => setEmailErr('')}
 					error={emailErr}
 				/>
 				<FormInput
 					title="Twitter"
 					text={twitter}
-					placeholder="@zangimmortal"
+					placeholder="zangimmortal"
 					onChangeText={handleChangeTwitter}
+					onFocus={() => setTwitterErr('')}
 					error={twitterErr}
-					onFocus={() => {
-						if (twitter.length === 0) setTwitter('@');
-					}}
-					onBlur={() => {
-						if (twitter === '@') setTwitter('');
-					}}
+					prefix={<Text style={styles.text}>@</Text>}
```

Comment:
with prefix, does request body of `joinWaitlist mutation` include `@` in twitter handle?